ssl-config/src/main/java/com/palantir/remoting/ssl/KeyStoreConfiguration.java

```diff
@@ -33,7 +32,7 @@
 public abstract class KeyStoreConfiguration {
 
     @Value.Parameter
-    public abstract Path path();
+    public abstract URI path();
```

Comment:
yeah it should be URI, and there should be some javadoc here to make it explicit that just entering the path is fine.